doc/aspice_swe3_4/example/iceoryx_component/test/moduletests/test_example_base_class.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/example_component/example_base_class.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+using ::testing::Return;
+
+namespace example
```

Comment:
not sure about this. Have you thought about `test::example`?

Reply:
We choosed this namespace order on purpose to have the test well sorted in doxygen by the component. Otherwise you have one big namespace test where all tests are lying and then its harder to figure it out on doxygen.